elements/lisk-bft/test/fixtures/accounts.ts

```diff
@@ -40,10 +40,8 @@ export const genesis = {
 export const mem_accountsFields = [
 	'username',
 	'isDelegate',
-	'secondSignature',
```

Comment:
Unless these fields are removed from the database table, they should not be excluded from these fixtures.